CHANGELOG.md

```diff
@@ -1,3 +1,244 @@
+## [Sidero 0.3.0-alpha.1](https://github.com/talos-systems/sidero/releases/tag/v0.3.0-alpha.1) (2021-05-20)
+
+Welcome to the v0.3.0-alpha.1 release of Sidero!  
+*This is a pre-release of Sidero*
+
+
+
+Please try out the release binaries and report any issues at
+https://github.com/talos-systems/sidero/issues.
+
+### ServerClass `any`
+
+Sidero now creates ServerClass `any` which matches all servers.
```

Comment:
Should we also mention `default` environment?

Reply:
Yes, we should, in general we should add to hack/release.toml as we introduce new features. As we missed that, we should do this as part of this PR